docs/integrations/pytorch_lightning.rst

```diff
@@ -1,318 +1,744 @@
+
 Neptune-PyTorch Lightning Integration
 =====================================
 
-The integration enables you to log |PyTorch Lightning| metrics to Neptune.
-
-.. image:: ../_static/images/integrations/pytorchlightning_neptuneml.png
-   :target: ../_static/images/integrations/pytorchlightning_neptuneml.png
+What will you get?
+------------------
+.. image:: ../_static/images/integrations/lightning_basic.gif
+   :target: ../_static/images/integrations/lightning_basic.gif
    :alt: PyTorchLightning neptune.ai integration
 
-Requirements
-------------
-This integration is enabled as part of PyTorch Ligthning loggers module, so all you need is to have |neptune-client| installed.
+PyTorch Lightning is a lightweight PyTorch wrapper for high-performance AI research. With Neptune integration you can:
 
-.. code-block:: bash
+* see experiment as it is running,
+* log training, validation and testing metrics, and visualize them in Neptune UI,
+* log experiment parameters,
+* monitor hardware usage,
+* log any additional metrics of your choice,
+* log performance charts,
+* save model checkpoints.
+
+Where to start?
+---------------
+To get started with this integration, follow whatever is most convenient to you:
+
+#. :ref:`Quickstart <quickstart>` below for detailed explanations,
+#. Open Colab notebook (badge-link below) with quickstart code and run it as a "`neptuner`" user - zero setup, it just works,
+#. View quickstart code as a plain Python script on |script|.
+
+You can also check this public project with example experiments: |project|.
+
+|Run on Colab|
+
+.. note::
+
+    This integration is tested with ``pytorch-lightning==0.9.0`` and ``neptune-client==0.4.122``.
+
+.. _quickstart:
+
+Quickstart
+----------
+This quickstart will show you how to log PyTorch Lightning experiments to Neptune using ``NeptuneLogger`` (part of the pytorch-lightning library).
+
+.. _before-you-start-basic:
+
+Before you start
+^^^^^^^^^^^^^^^^
+You have ``Python 3.x`` and following libraries installed:
+
+* ``neptune-client==0.4.122`` or newer: See :ref:`neptune-client installation guide <installation-neptune-client>`.
+* ``pytorch`` and ``torchvision``. See |pytorch-install|.
+* ``pytorch-lightning==0.9.0``. See |lightning-install|.
+
+You also need minimal familiarity with the PyTorch Lightning. Have a look at the "|lightning-guide|" guide to get started.
 
-    pip install neptune-client
+Step 1: Import Libraries
+^^^^^^^^^^^^^^^^^^^^^^^^
+Import necessary libraries.
 
+.. code-block:: python3
 
-Create the **LightningModule**
-------------------------------
+    import os
 
-Nothing changes here, just your typical ``LightningModule`` structure.
+    import torch
+    from torch.nn import functional as F
+    from torch.utils.data import DataLoader
+    from torchvision.datasets import MNIST
+    from torchvision import transforms
+
+    import pytorch_lightning as pl
+
+Notice ``pytorch_lightning`` at the bottom.
+
+Step 2: Define Hyper-Parameters
```

Comment:
I started wondering. Should this be the basic usage or advanced use already. 
What do you think?

Reply:
We talked about it :)